README.md

```diff
@@ -22,14 +22,15 @@ Versions / Roadmap
 
 Tron is changing and under active development.
 
-It is being transformed from an ssh-based execution engine to a [Mesos
-framework](http://mesos.apache.org/documentation/latest/frameworks/).
+It is being transformed from an ssh-based execution engine to a [Kubernetes
+framework](https://kubernetes.io/docs/concepts/overview/).
 
 Tron development is specifically targeting Yelp's needs and not designed to be
 a general solution for other companies.
 
 * <= v0.6.2 - Stable version, recommended for non-Yelp installations.
-* >= v1.0.0 - Stable mesos-based version
+* \>= v1.0.0 & <= v1.32.0  - Stable mesos-based version
+* \> v1.32.0 - Stable Kubernetes version
```

Comment:
imo, it'd be fine to remove these versions recommendations :)

(we're really just developing in the open, i don't think we're really assuming that non-yelpers will ever use this :p)